model/trace/messaging.yaml

```diff
@@ -164,3 +164,11 @@ groups:
         tag: tech-specific-gcp-pubsub
         requirement_level:
           conditionally_required: If the message type has an ordering key set.
+      - ref: messaging.gcp_pubsub.message.delivery_attempt
+        tag: tech-specific-gcp-pubsub
+        requirement_level: required
```

Comment:
for HTTP we only require a similar attribute (`http.request.resend_count`) if and only if it's more than 0. I.e. first try/attempt does not have this attribute.
This reduces telemetry volume which is always a good thing.

I suggest doing the same here, please use `http.request.resend_count` as an example

Reply:
I think this is slightly different, since it is only included when a subscription has exactly once delivery enabled (but @hongalex can correct me if this is wrong) so we would like to make a separate attribute.